javaScript.js

```diff
@@ -123,6 +123,26 @@ function closeMenu() {
   document.getElementById('my-menu').style.height = '0%';
 }
 
+/* -- Email Validation -- */
+const form = document.getElementById('contactForm');
+const email = document.getElementById('email');
+const alert = document.getElementById('alertWin');
+email.addEventListener = ('keyup', () => { alert.style.display = 'none'; });
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  const emailVal = email.value.trim();
+
+  if (!emailRegex.test(emailVal)) {
+    email.focus();
+    alert.style.display = 'block';
+  } else {
+    form.submit();
+  }
+});
```

Comment:
While you did a good job trying to implement the form validation you missed out a few pointers. I'm saying this because your form is still send to form spree with upper case email. An error message is supposed to pop up if font is in Upper case. Please try this example it may be of help:

Example: 
window.onload = showWorks();
const formContainer = document.querySelector('.con-form');
formContainer.addEventListener('submit', (e) => {
  e.preventDefault();
  const mailInput = document.getElementById('mail');
  const messageError = document.querySelector('.error-message');

  if (mailInput.value === mailInput.value.toLowerCase()) {
    messageError.textContent = '';
  } else {
    messageError.innerHTML = '*email must be in lower case <br> * form not sent';
  }
});